src/WinRPM.jl

```diff
@@ -458,7 +458,7 @@ function do_install(package::Package)
         error("failed to download $name $(data[2]) from $source/$path.")
     end
     cache = getcachedir(source)
-    path2 = joinpath(cache,escape(path))
+    path2 = joinpath(cache, match(r"[^/]+$",path).match)
```

Comment:
The `match` should probably be replaced with `basename`

Reply:
Did not know about that function! That seems to do the trick just fine.